### PR TITLE
Added command for ZG-204ZM battery human millimeter wave

### PR DESCRIPTION
**Pull request by 13717033460**
Added command for ZG-204ZM battery human millimeter wave

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -6296,6 +6299,11 @@ const definitions: Definition[] = [
                 [2, 'static_detection_sensitivity', tuya.valueConverter.raw],
                 [107, 'indicator', tuya.valueConverter.onOff],
                 [121, 'battery', tuya.valueConverter.raw],
+                [122, 'motion_detection_mode', tuya.valueConverterBasic.lookup({
+                    'Only PIR': tuya.enum(0), 'PIR+Dadar': tuya.enum(1), 'Only Dadar': tuya.enum(2),
```
Comment (Koenkk) [inline src/devices/tuya.ts:6303]:
```suggestion
                    'only_pir': tuya.enum(0), 'pir_and_dadar': tuya.enum(1), 'only_dadar': tuya.enum(2),
```

File: src/devices/tuya.ts [hunk 2]
```diff
@@ -6283,6 +6283,9 @@ const definitions: Definition[] = [
             e.numeric('static_detection_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(1).withUnit('x')
                 .withDescription('Static detection sensitivity'),
             e.binary('indicator', ea.STATE_SET, 'ON', 'OFF').withDescription('LED indicator mode'),
+            e.enum('motion_detection_mode', ea.STATE_SET, ['Only PIR', 'PIR+Dadar', 'Only Dadar']).withDescription('Motion detection mode'),
```
Comment (Koenkk) [inline src/devices/tuya.ts:6286]: Also update the values here.

Comment (13717033460) [reply src/devices/tuya.ts:6286]: ok,thank you

Comment (Koenkk) [reply src/devices/tuya.ts:6286]: Could you update it?

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**